### ISSUE TITLE
Suggestion: Only accept screenshots if their resolutions are at least 100x100

### ISSUE THREAD

**Issue by DandelionSprout** [AdguardTeam/ReportsWebApp#331]
It would've hopefully led to reports like https://github.com/AdguardTeam/AdguardFilters/issues/153931 not using nonsense "screenshots".

So the suggestion I came up with in a hurry, would be for the "Upload screenshot" section to reject images that are not at least 100px tall and 100px wide.

**Comment by Alex-302** [comment]
Very rare case. Sometimes it is not required at all.

**Comment by DandelionSprout** [comment]
How about "Less than 50x50", then?

**Comment by Alex-302** [comment]
@DandelionSprout In this case user may upload a random image. There have already been cases of uploading *kpicks)

**Comment by DandelionSprout** [comment]
Good point, actually; which I personally suppose crosses over into https://github.com/AdguardTeam/ReportsWebApp/issues/349

**Comment by Alex-302** [comment]
> #349

it is just a bug
